.github/workflows/tests.yml

```diff
@@ -90,7 +91,7 @@ jobs:
         run: docker exec builder bash -c "cd FlexibleSUSY && for m in $MODELS; do ./createmodel --with-math-cmd=/usr/local/Wolfram/WolframEngine/12.1/Executables/math --name=\$m; done"
 
       - name: Configure
-        run: docker exec builder bash -c "cd FlexibleSUSY && if [ \"$LOOPLIBRARY\" = \"looptools\" ]; then export FL=\"--enable-looptools --with-looptools-libdir=/LoopTools-$LOOPTOOLS_VERSION/lib64 --with-looptools-incdir=/LoopTools-$LOOPTOOLS_VERSION/include\"; fi && ./configure --with-cxx=$CXXCOMPILER --with-math-cmd=/usr/local/Wolfram/WolframEngine/12.1/Executables/math --with-optional-modules=test --with-models=\$(echo $MODELS| tr ' ' ',') \$FL"
+        run: docker exec builder bash -c "cd FlexibleSUSY && if [ \"$LOOPLIBRARY\" = \"looptools\" ]; then export FL=\"--with-loop-libraries=looptools --with-looptools-libdir=/LoopTools-$LOOPTOOLS_VERSION/lib64 --with-looptools-incdir=/LoopTools-$LOOPTOOLS_VERSION/include\"; elif [ \"$LOOPLIBRARY\" = \"collier\" ]; then export FL=\"--with-loop-libraries=collier --with-collier-libdir=/COLLIER/lib --with-collier-incdir=/COLLIER/include\"; fi && ./configure --with-cxx=$CXXCOMPILER --with-math-cmd=/usr/local/Wolfram/WolframEngine/12.1/Executables/math --with-optional-modules=test --with-models=\$(echo $MODELS| tr ' ' ',') \$FL"
```

Comment:
Is it possible to write this long line over multiple lines? I'm not smart enough to read this. :)

Reply:
Maybe. I'm constantly thinking about it. The core problem is that we need to run the docker image explicitly. And then we run every command from the base image on docker image as `docker exec builder bash -c "..."`. A much nicer solution is telling in an action to directly run every command in the container as
```
runs-on: ubuntu-18.04
container: navir/opensuseleap-for-flexiblesusy
```
The problem is with mathematica. The Wolfram Engine then rejects my license. I don't know why and whether it's possible to fix it at all. I can reduce the length of the line a little even without it, though.

Reply:
I've played around a bit and found a site to convert YAML to JSON. It tells me that when I write
~~~
run:
  - >
    cmd "if [ -n "$file" ] ; then
    echo 'Hi there!' ;
    fi " | docker -
  - other_cmd
~~~
it gets translated to
~~~
{
   "run": [
      "cmd \"if [ -n \"$file\" ] ; then echo 'Hi there!' ; fi \" | docker -\n",
      "other_cmd"
   ]
}
~~~
So, I think it is possible to split the long line, even in the middle of the long string command that you're passing to docker.

Reply:
I've split the line in b8d9a8032cc60fa57a12035feb52b5f995bf7405. Let me know if this is what you wanted.

Reply:
Many thanks, this looks much better! :)